demo/index.html

```diff
@@ -61,11 +61,33 @@
         });
         pv.loadPanorama(lat, lon);
       }
+      
+      // Load with URL Params lat & lon e.g. https://www.yoursite.com/panoviewer/index.html?lat=52.302919&lon=4.857236
+      function getUrlParameter(name) {
+        name = name.replace(/[\[]/, '\\[').replace(/[\]]/, '\\]');
+        var regex = new RegExp('[\\?&]' + name + '=([^&#]*)');
+        var results = regex.exec(location.search);
+        return results === null ? '' : decodeURIComponent(results[1].replace(/\+/g, ' '));
```

Comment:
Waarom niet iets als:
```
let params = {};
location.search.substring(1).split('&').forEach((entry) => {
  const [key, value] = entry.split('=');
  params[decodeURIComponent(key)] = decodeURIComponent(value);
});
return params;
```
Misschien wat ES6 dingen in ES5 schrijven, maar zo heb je makkelijk alle search parameters in een object zitten.